src/components/organisms/UiTabs/UiTabs.vue

```diff
@@ -75,32 +77,61 @@ const itemsToRender = computed<TabsProps['items']>(() => (props.items.map((item,
     name: name || `tabs-item-${key}`,
   };
 })));
-const tabs = ref<HTMLDivElement | null>(null);
-const activeTabHTMLElement = ref<HTMLElement | null>(null);
+
+const tabs = ref<OrNull<HTMLDivElement>>(null);
+const activeTabEl = ref<OrNull<HTMLElement>>(null);
+
+const containerX = ref(0);
+const containerY = ref(0);
+const containerH = ref(0);
+
 const offsetX = computed(() => {
-  if (activeTabHTMLElement.value === null) return 0;
+  const hack = containerX.value * 0;
```

Comment:
I defined this hacky variable to make computed properties react to changes from resizeObserver.

Reply:
@KamilBartczak03 I'd consider creating `watch` with multiple sources that modify `ref` value instead of this hack.
Watchers require more code, but it will be more obvious in terms of extending and debugging.